### PR TITLE
README Update 1

### PR DESCRIPTION
**Pull request by TonyDiMaggio**
Update 1 of README. Please check for spelling errors, grammatical errors, and inaccuracies in regards to features and technologies. Thank you!

[AB#135](https://dev.azure.com/dhk2/65e21a49-896d-470d-8659-fc7cca2f4625/_workitems/edit/135)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,80 @@
-# TheBugTracker
-CAPSTONE BUG TRACKER PROJECT
+# Autobug
```
Comment (DanaKhoshnaw) [inline README.md:1]: Autobug Tracker?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,80 @@
...
+## Contributors
+### Product Owners / Sponsors
+> [Daniel Connelly](https://www.linkedin.com/in/dconnelly2/)
+> [Teal Dulcet](https://www.tealdulcet.com/)
+
+### Development Team
+#### Team Lead
+> Antonio DiMaggio
+
+#### Software Engineers
```
Comment (DanaKhoshnaw) [inline README.md:74]: Just a suggestion, reorder the name to be alphabetically either by first or last name

File: README.md (hunk 1 above)
Comment (TonyDiMaggio) [reply README.md:1]: Oh yeah, your right. AutoBugTracker is what we called it with Daniel.

File: README.md (hunk 2 above)
Comment (TonyDiMaggio) [reply README.md:74]: will do.

**Pull request merged**